schema/defs/Variation.rst

```diff
@@ -16,9 +16,9 @@ A representation of the state of one or more biomolecules.
       - Description
    *  - _id
       - :ref:`CURIE`
-      - 0..1
+      - 1..1
```

Comment:
`_id` is optional. Should be `0..1`, right?

Reply:
Oh wow. Thanks @reece, great catch. Will fix.